### ISSUE TITLE
Mongoose in docs

### ISSUE THREAD

**Issue by Dvoikin** [pip-services3-dotnet/pip-services3-mongodb-dotnet#2]
![image](https://user-images.githubusercontent.com/7523735/96741984-12d26800-13cb-11eb-8dea-7c328b5c103f.png)

Mongoose is part of Node.js implementation only. Correct me pls if I am not right else I guess we would remove it from the documentation in all languages and platforms except Node.js. 

Found this here: https://github.com/pip-services3-dotnet/pip-services3-mongodb-dotnet/blob/master/src/Persistence/MongoDbPersistence.cs

**Comment by markmak77** [comment]
I checked dotnet, python, go, dart, and java - found this copy-paste mistake only in dotnet and python. All of the other languages didn't have this typo. 

For dotnet, I removed all Mongoose mentions from the code and ran Doxygen (using the project's Doxyfile) to re-generate the documentation. For python, I removed all Mongoose mentions from the code and from the htmls in the doc folder (didn't re-generate using Sphinx).

I also noticed that both projects have duplicate doc/docs folders. In dotnet I tried to just remove the duplicate "docs" folder, but it didn't pass the pipeline. We should find out why and get this cleaned up as well (in both python and dotnet).
